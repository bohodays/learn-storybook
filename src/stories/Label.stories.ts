import type { Meta, StoryObj } from "@storybook/react-vite";

import { fn } from "storybook/test";
import Label from "../components/Label";

// Docs 정보
const meta = {
  title: "Text/Label", // 경로를 나타냄
  component: Label,
  parameters: {
    layout: "centered",
  },
  tags: ["autodocs"],
  argTypes: {
    htmlFor: { control: "text", desciption: "label의 for 속성" },
    children: { control: "text", description: "label의 내용" },
  },
} satisfies Meta<typeof Label>;

export default meta;
type Story = StoryObj<typeof meta>;

export const Default: Story = {
  args: {
    htmlFor: "username",
    children: "이메일",
  },
};
