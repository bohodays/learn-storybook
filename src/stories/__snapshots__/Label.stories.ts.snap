// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text/Label Default smoke-test 1`] = `
<label class="text-sm text-primary"
       for="username"
>
  이메일
</label>
`;
